contracts/src/deployers/reth/RETHHyperdriveDeployerCoordinator.sol

```diff
@@ -52,15 +47,7 @@ contract RETHHyperdriveDeployerCoordinator is HyperdriveDeployerCoordinator {
             _target3Deployer,
             _target4Deployer
         )
-    {
-        rocketStorage = _rocketStorage;
-
-        // Fetching the RETH token address from the storage contract.
-        address rocketTokenRethAddress = _rocketStorage.getAddress(
```

Comment:
rETH contract is not upgradable per docs so safe to make constant https://docs.rocketpool.net/developers/usage/contracts/contracts